tox.ini

```diff
@@ -5,6 +5,7 @@ envlist = py27,py35,py36
 deps =
     pytest
     pytest-cov
+    parsel
```

Comment:
I wonder if we should add another environment without ``parsel`` - since we don't require it's installation, we need to make sure that functions such as ``html_text.extract_text`` do not import it and work find without it. Right now we don't check it in tests, and can only check by reading the code.